cmd/login.go

```diff
@@ -20,22 +20,28 @@ import (
 
 //Login starts the login handshake with github and okteto
 func Login() *cobra.Command {
+	showUrl := false
 	cmd := &cobra.Command{
 		Use:   "login [url]",
```

Comment:
Maybe some improvement in usage could be helpful? But how would that look like?
`Use:   "login [url|'showUrl']",`

Reply:
I think we should always show the URL and not have the option. This is a pattern that other CLIs use as well.

Reply:
alrights. sound better.